### DISCUSSION TITLE
Samples with Microsoft.AspNetCore.Mvc.Testing

### DISCUSSION THREAD

**Discussion by Kralizek** [dodyg/practical-aspnetcore#280] [General]
Is there any interests in samples showing how to use `Microsoft.AspNetCore.Mvc.Testing` to test ASP.NET Core applications?

If so, I could contribute a sample testing a classic web application later this week or the next.

**Comment by dodyg** [comment]
That would be awesome. We currently have not touched testing at all.

**Reply by Kralizek** [reply]
I can put something together :)

Should I create an issue pointing to this discussion?

**Reply by dodyg** [reply]
Yup

**Reply by adlansyiari73** [reply]
How make connection from asp.net to mysql ? Can give me tutorial?
